tools/MUXPGODatabase/template.ps1

```diff
@@ -0,0 +1,16 @@
+function replace_many ( $string, $dictionary )
```

Comment:
The convention for powershell functions is: Verb-Noun, e.g.
FillOut-Template

Reply:
Fixed